BenadrylAutomation/src/main/java/com/jnj/pageobjects/ProductPage.java

```diff
@@ -0,0 +1,284 @@
+
+	@FindBy(id = "tab_warnings")
+	private WebElement warningsHeading;
+
+	@FindBy(id = "mini-panel-product_faq")
+	private WebElement faqsSection;
+
+	@FindBy(css = "#mini-panel-product_faq span")
+	private WebElement faqsHeading;
+
+	@FindBy(css = "#bv-product-reviews")
+	private WebElement reviewsSection;
+	
+	/**
+	 * Returns the WebElement of a 'Jump To' heading based on its text.
+	 *
+	 * @param headingText The text of the 'Jump To' heading.
+	 * @return WebElement The WebElement of the specified 'Jump To' heading.
+	 */
+	public WebElement getProductName(String path) { 
+		 return	 driver.findElement(By.xpath("//span[@class='node__title']//a[@href='/products/" + path + "']"));
+	}	
+	
+	/**
+	 * Returns the WebElement of a 'Jump To' heading based on its text.
+	 *
+	 * @param headingText The text of the 'Jump To' heading.
+	 * @return WebElement The WebElement of the specified 'Jump To' heading.
+	 */
+	public WebElement getJumpToHeadings(String headingText) { 
+		 return	 driver.findElement(By.xpath("//div[@class='jump-to-wrapper']//ul//li//a[text()='" + headingText + "']"));
+	}
+	
+	/**
+	 * Returns the WebElement of the Write a Review modal.
+	 */
+	public WebElement getWriteAReviewHeading() { 
+		 return	 driver.findElement(By.id("shadow-root")).getShadowRoot().findElement(By.cssSelector("#bv-ips-loading-text_bv-ips-title > span.ips__sc-r8pfgm-3.fINXNs"));
+	}
+	
+	/**
+	 * Navigates to a product detail page from the homepage by clicking on the product header menu 
+	 * and a specific product title.
+	 * 
+	 * @param link of the product url
+	 */
+	public void visitDetailsPage(String path) {
+	    Action.performActionwithExtentInfoLog(prodHeaderMenu, "click", "Clicking on : " + prodHeaderMenu.getText());
+	    Action.performActionwithExtentInfoLog(getProductName(path), "click", "Clicking on : " + prodTileTitle.getText());
+	    Action.verifyPageUrl("products/" + path);
+	}
+	
+	/**
+	 * Navigates to a product detail page from the homepage by clicking on the product header menu 
+	 * and a specific product title.
+	 */
+	public void visitPDP() {
+	    Action.performActionwithExtentInfoLog(prodHeaderMenu, "click", "Clicking on : " + prodHeaderMenu.getText());
+	    Action.performActionwithExtentInfoLog(prodTileTitle, "click", "Clicking on : " + prodTileTitle.getText());
+	}
+
+	/**
+	 * Verifies that the product image is visible on the product detail page.
+	 */
+	public void verifyProdImage() {
+	    extentInfoLog("Image is displayed : ", Action.isDisplayed(driver, prodImg));
+	}
+
+	/**
+	 * Verifies that the product title is visible on the product detail page.
+	 * @param expectedTitle The text expected to be found on the title of the product page.
+	 */
+	public void verifyProdTitle(String expectedTitle) {
+	    this.visitPDP();
+	    String actualText = prodTitle.getText();
+	    Action.printAndAssert(actualText, expectedTitle);
+	    extentInfoLog("Title is displayed : ", Action.isDisplayed(driver, prodTitle));
```

Comment:
@zahinSJI no need of this. remove this line.

Reply:
Removed